sdk/identity/Azure.Identity/src/AggregateAuthenticationException.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using System;
+using System.Text;
+
+namespace Azure.Identity
+{
+    /// <summary>
+    /// An exception which is raised when multiple <see cref="TokenCredential"/> fail to authenticate, such as when invoked by the <see cref="ChainedTokenCredential"/> or the <see cref="DefaultAzureCredential"/>
+    /// </summary>
+    internal class AggregateAuthenticationException : AuthenticationFailedException
```

Comment:
Why did you decide not to use an existing aggregate exception type?

Reply:
I did, I was just using this as a builder essentially, but I can remove the class altogether if people strongly object to the internal exception type.

Reply:
Removed